frontend/pages/schedule/ManageSchedulePage/ManageSchedulePage.tsx

```diff
@@ -43,9 +43,10 @@ const renderTable = (
   isOnGlobalTeam: boolean,
   selectedTeamData: ITeam | undefined,
   isFetchingGlobalScheduledQueries: boolean,
-  isLoadingTeamScheduledQueries: boolean
+  isLoadingTeamScheduledQueries: boolean,
+  errorQueries: Error | null
 ): JSX.Element => {
-  return allScheduledQueriesError ? (
+  return allScheduledQueriesError || errorQueries ? (
```

Comment:
need errorQueries because Schedule a query butto rendered requires access to queriesAPI